js/base/Exchange.js

```diff
@@ -693,8 +697,11 @@ module.exports = class Exchange {
         currencies.forEach (currency => {
 
             if (typeof balance[currency].used === 'undefined') {
+                // exchange reports only 'free' balance -> try to derive 'used' funds from open orders cache
 
-                if (this.parseBalanceFromOpenOrders && ('open_orders' in balance['info'])) {
```

Comment:
Oops, looks like we actually have an exchange-specific fieldname (`open_orders` is Liqui-only) in the base class. Will need to rethink this codepath a little to move the exchange-specific code to the derived class...

Reply:
Yeah, I noticed that too. Still, don't know what exchanges are affected (beyond liqui) so left it as is.

Reply:
To be more precise, I've seen balance accompanied by various account info on a number of exchanges. If we dedicate some property in `balance` object to be responsible for holding such info I'll move it's population to an exchange class.

Reply:
Yep, moreover, we are also considering making use of the balances returned from *order* APIs (liqui does return renewed balanced on placing orders, some other exchanges do that as well). This is the reason for reserving a `.balance = {}` property on the exchange instance for further caching needs. We actually think of adding a caching layer to balances as well, however, this requires a significant rework of the entire cache (to make it robust and intuitive)...